kalman_watch/options.py

```diff
@@ -0,0 +1,55 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Watch Kalman star data during perigee passages."""
+
+import os
+from pathlib import Path
+
+import astropy.config as _config_
+
+
+class ConfigNamespace(_config_.ConfigNamespace):
+    rootname = 'ska_trending'
```

Comment:
The reason why I use this name is because I was thinking things would slowly converge into a single package (like Jean's ska_trend), and I would rather have one single configuration file rather than a collection of files.

I can change it back to use `kalman_watch` if that is what you really want.

I would note that I did experiment creating config files and it worked. I did see a bug where the file was not created (like your issue). I would have to check again what were the circumstances, but the name was not the cause of the misbehavior.